sdk/iOS/test/MSCoreDataStoreTests.m

```diff
@@ -47,6 +50,54 @@ -(void)testInit
     XCTAssertNotNil(self.store, @"store creation failed");
 }
 
+// Tests MSCoreDataStore Read, Update and Delete operations are case insensitive for the Id column.
+-(void)testReadUpdateDelete_CaseInsensitiveIdOperations
+{
+    NSError *error;
+
+    // Seed data to be inserted in the table
+    NSString * const originalItemId = @"itemid";
```

Comment:
Style guide is *const, no space.  If we are keeping them, when are we proposing to use them?  I notice the syncTable and the query are not using it.  Its a lil ambiguous and it doesn't line up with patterns we've used throughout the test suite.